### PR TITLE
Refactor path for configuration file, option(s) and Docker image

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

File: service/init.go [hunk 1]
```diff
@@ -98,30 +95,30 @@ func (*SMF) GetCliCmd() (flags []cli.Flag) {
...
-		if err := factory.InitRoutingConfigFactory(DefaultUERoutingPath); err != nil {
-			return err
-		}
+	if err = factory.InitConfigFactory(absPath); err != nil {
+		return err
+	}
+
+	factory.SmfConfig.CfgLocation = absPath
+
+	ueRaoutingPath, err := filepath.Abs(config.uerouting)
```
Comment (thakurajayL) [inline service/init.go:114]:
```suggestion
	ueRoutingPath, err := filepath.Abs(config.uerouting)
```

File: service/init.go [hunk 2]
```diff
@@ -98,30 +95,30 @@ func (*SMF) GetCliCmd() (flags []cli.Flag) {
...
+	if err != nil {
+		logger.CfgLog.Errorln(err)
+		return err
+	}
+
+	if err := factory.InitRoutingConfigFactory(ueRaoutingPath); err != nil {
```
Comment (thakurajayL) [inline service/init.go:120]:
```suggestion
	if err := factory.InitRoutingConfigFactory(ueRoutingPath); err != nil {
```

File: service/init.go (hunk 1 above)
Comment (gab-arrobo) [reply service/init.go:114]: Done

File: service/init.go (hunk 2 above)
Comment (gab-arrobo) [reply service/init.go:120]: Done

**Pull request merged**